airflow/contrib/operators/databricks_operator.py

```diff
@@ -163,9 +173,31 @@ def __init__(
         if 'run_name' not in self.json:
             self.json['run_name'] = run_name or kwargs['task_id']
 
+        self.json = self._deep_string_coerce(self.json)
         # This variable will be used in case our task gets killed.
         self.run_id = None
 
+    def _deep_string_coerce(self, content, json_path='json'):
+        """
+        The ``self.json`` field must be a dict with only string values. This is because
+        ``render_template`` will fail for numerical values.
+        """
+        c = self._deep_string_coerce
+        if isinstance(content, six.string_types):
+            return content
+        elif isinstance(content, six.integer_types+(float,)):
+            return str(content)
```

Comment:
Does converting numbers into strings introduce any type issues? Probably need an inline comment to assure readers that this is safe.